lib/omniauth/strategies/oidc/back_channel_logout_request.rb

```diff
@@ -0,0 +1,81 @@
+module OmniAuth
+  module Strategies
+    class Authentiq
+      class BackChannelLogoutRequest
+
+        def initialize(strategy, request)
+          @strategy, @request = strategy, request
+        end
+
+        def call(options = {})
+          @options = options
+
+          begin
+            # for
+            # sign_out_callback.call(*back_channel_logout_request)
+            # to actually happen
+            # a proc must be set in the devise.rb initializer of the rails app
+            result = sign_out_callback.call(*back_channel_logout_request)
+          rescue
+            return backchannel_logout_response(400, ['Bad request']).finish
```

Comment:
I'd prefer to change the body content to the exception description if possible, so that also JWT exceptions (e.g. `raise JWT::VerificationError, 'Algorithm not supported'`) come through and are presented to user.

The "Bad request" body is redundant anyway since that is already in the HTTP status line.